source/includes/_hoist.md

```diff
@@ -0,0 +1,55 @@
+# Hoist
+
+```javascript
+1.
+var name = "Warren";
+//Before Hoist
+
+var name; // This is the declaration
+name = "Warren"; //This is the assignment
+//After Hoist
+
+```
+
+
+
+```javascript
+ 2.
+function parkour(){} //Hoisted function declaration
+function iDeclareBankruptcy(){} //Hoisted function declaration
+
+var parkour; //Hoisted variable declaration
+var iDeclareBankruptcy(){} //Hoisted variable declaration
+
+parkour = function(){} // Function expression
+iDeclareBankruptcy = function(){} // Function expression
+
+```
+
+__1.__
+
+* Hoisting is when JavaScript __splits__ the variable and function declarations and moves them to the top of the container scope
+, while retaining the same order.
+
+* Hoisting is made to organize code and used for the browser to understand
+function and variable declarations before page is fully loaded.
+
+* However the assignment part of the variable remains unmoved.
```

Comment:
Assignment is not 'part' of a variable. 

If we have:
```js
var foo = 'bar';
```

We have the variable declaration:

```js
var foo;
```

And the assignment operation:

```js
foo = 'bar';
```